CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ this project uses date-based 'snapshot' version identifiers.
 
 ## [Unreleased]
 
+### Changes
```

Comment:
As we don't release `CONTRIBUTING`, changes here don't go into the log.

Reply:
CONTRIBUTION _is_ packed and uploaded to CTAN, see https://mirrors.rit.edu/CTAN/macros/latex/contrib/l3build/CONTRIBUTING.md

Reply:
Er, that's an oversight we should fix :)

Reply:
(Probably easiest is moving it to `.github` once we merge your Pr)